sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusSenderClientTest.java

```diff
@@ -144,6 +150,38 @@ void createsMessageBatchWithSize() {
         Assertions.assertEquals(batch, messageBatch);
     }
 
+    /**
+     * Verifies that sending an array of message will result in calling sender.send(Message...).
+     */
+    @Test
+    void sendMessageList() {
+        // Arrange
+        final int count = 4;
+        final byte[] contents = TEST_CONTENTS.getBytes(UTF_8);
+        final ServiceBusMessage[] messages = new ServiceBusMessage[count];
+
+        IntStream.range(0, count).forEach(index -> {
```

Comment:
Why not `for (int i = 0; i < messages.length; i++)`? I don't see any added benefit creating this IntStream.

Reply:
There's a helper method that does all this: `TestUtils.getServiceBusMessages(3, id).toArray()`